Combine scenarios for MS Word workload

muendelezaji opened the pull request:
Fixes #89

jimboatarm commented on wlauto/workloads/msword/uiauto/src/com/arm/wlauto/uiauto/UiAutomation.java:
```diff
@@ -80,17 +78,15 @@ public void runUiAutomation() throws Exception {
         clickUiObject(BY_TEXT, "Skip", true); // skip welcome screen
         stopLogger("welcome_screen_skip");
+        // Run the main tests
+        testExistingDocument(parameters.getString("document_name"));
```
See comment in the other pull request.  The agenda should pass a param to enable the testing of an existing doc.  Then this workload will work OOB for typical WA users.

muendelezaji commented:
> See comment in the other pull request #88. The agenda should pass a param to enable the testing of an existing doc. Then this workload will work OOB for typical WA users.

Updated. Uses a flag `use_test_file` which is disabled by default. Checks that a parameter `test_file` is provided when the flag is enabled.
Also cleaned up some test tags for better consistency
